docs/io/ascii/index.rst

```diff
@@ -229,6 +229,63 @@ ASCII with no loss of information.
 ..
   EXAMPLE END
 
+Miscellaneous Examples
```

Comment:
This should be a subsection of https://astropy.readthedocs.io/en/latest/io/ascii/read.html#bad-or-missing-values and should be titled `Selecting columns for masking`.

Reply:
Sure, have moved the changes to the [io/ascii/read.html#bad-or-missing-values](https://astropy.readthedocs.io/en/latest/io/ascii/read.html#bad-or-missing-values) section instead.